dns/route53/dns_test.go

```diff
@@ -4,77 +4,55 @@ import (
 	"testing"
 
 	"net"
+	"fmt"
+	"os"
 
 	"github.com/aws/aws-sdk-go/aws"
-	route53Client "github.com/aws/aws-sdk-go/service/route53"
 	log "github.com/sirupsen/logrus"
 	"github.com/stretchr/testify/assert"
 )
 
-func TestEditResourceRecords(t *testing.T) {
+func TestAddAndRemove(t *testing.T) {
 	log.SetLevel(log.DebugLevel)
+
+	rootFqdn := os.Getenv("TEST_ZONE_NAME")
```

Comment:
This is a unit test, not an integration test. It shouldn't need to do any network access, or be influenced by environment variables.